packages/react-devtools-shared/src/devtools/views/hooks.js

```diff
@@ -239,7 +239,9 @@ export function useModalDismissSignal(
       ownerDocument = ((modalRef.current: any): HTMLDivElement).ownerDocument;
       ownerDocument.addEventListener('keydown', handleDocumentKeyDown);
       if (dismissOnClickOutside) {
-        ownerDocument.addEventListener('click', handleDocumentClick, true);
+        ownerDocument.addEventListener('click', handleDocumentClick);
+        ownerDocument.removeEventListener('click', handleDocumentClick,true);
+
```

Comment:
Sorry. This is not what I meant.

We add event listeners in the effects body and we remove them in the cleanup function the effect returns.

We need to add the new `useCapture` param to _both places_ not just `addEventListener`

Reply:
```suggestion
        ownerDocument.addEventListener('click', handleDocumentClick, true);
```

Reply:
ok @bvaughn  i closed this one as i messed up the repo will be pulling another pr now